app/uk/gov/hmrc/nisp/views/includes/helpers/error_overview.scala.html

```diff
@@ -0,0 +1,16 @@
+@(form: Form[_])
+
+@if(form.hasErrors) {
+        <div id="error-summary-display" class="flash error-summary error-summary--show " role="group" aria-labelledby="error-summary-heading" tabindex="-1">
+            <h2 id="error-summary-heading" class="h3-heading">@Messages("nisp.validation.summary.heading")</h2>
+            <ul class="js-error-summary-messages">
+                @form.errors.map { error =>
+                <li role="tooltip">
+                    <a href='#@error.key' id='@error.key-error-summary' data-focuses='@error.key'>
+                        @Messages(error.message, error.args : _*).replaceAll("""[\.\[\]]""", "")
```

Comment:
I've removed regex and replace it by validation level, can you check it is correct now. Thanks